ice/settings.py

```diff
@@ -1,22 +1,28 @@
+import json
+
 from os import environ
 from pathlib import Path
+from typing import Any
+from typing import Optional
 from typing import TYPE_CHECKING
 
 from pydantic import BaseSettings
 
 if TYPE_CHECKING:
     AnyHttpUrl = str
-
 else:
     from pydantic import AnyHttpUrl
 
+from .logging import log_lock
```

Comment:
Please move this up so that normal imports stay together.

Reply:
ty! d00754e5ce56d2b3892ebb1f28a1de12cf11bf53